doc/source/quickstart.rst

```diff
@@ -70,25 +40,49 @@ scripts and XML files to configure a case (see below) for the requested
 resolution, component set, and machine. **create_newcase** has several
 required arguments (invoke **create_newcase --help** for help).
 
-If running on a supported machine, (``$MACHINE``), that machine will
+The ``$PROJECT`` variable must be specified in either your 
+shell environment or as the ``--project`` argument to **create_newcase**.
+
+If running on a supported machine, that machine will
 normally be recognized automatically and therefore it is not required
 to specify the ``--machine`` argument to **create_newcase**, then
```

Comment:
I'd split this into two sentences: "Then invoke create_newcase as follows:" should be its own sentence.

Reply:
fixed.